Library/Formula/truncate.rb

```diff
@@ -0,0 +1,39 @@
+class Truncate < Formula
+  desc "truncates a file to a given size"
+  homepage "http://www.vanheusden.com/truncate"
+  url "http://www.vanheusden.com/truncate/truncate-0.8.tgz"
+  sha256 "165914d643351324d69096427d3ce5d3ab13aed3f7c990a71ba3bd0c6f278f00"
+
+  patch :DATA # remove missing file; rename syscalls for OSX which are natively 64bit
```

Comment:
The developer is @flok99 on Github. But `truncate` is not in his public repos.  
https://github.com/search?q=user%3Aflok99+truncate&type=Code